app/Console/Commands/WaitingLists/ProcessPendingRemovals.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands\WaitingLists;
+
+use App\Console\Commands\Command;
+use App\Events\Training\AccountRemovedFromWaitingListDueToActivity;
+use App\Events\Training\AccountWithinFiveDaysOfWaitingListRemoval;
+use App\Models\Training\WaitingList;
+use Carbon\Carbon;
+
+class ProcessPendingRemovals extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'waitinglists:processpendingremovals';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Processing waiting list accounts marked for automatic removal';
+
+    /**
+     * Executes all necessary console commands.
+     */
+    public function handle()
+    {
+        WaitingList::all()->filter(function ($waitingList) {
+            return $waitingList->isAtcList();
+        })->each(function (WaitingList $waitingList) {
+            $waitingList->accounts
+            ->filter(function ($account) {
+                return $account->pivot->current_status->name == 'Active';
+            })->filter(function ($account) {
+                return $account->pivot->pending_removal?->status == 'Pending';
+            })->each(function ($account) use ($waitingList) {
+                // Send 5 day reminders
+                if (
+                    Carbon::parse($account->pivot->pending_removal->removal_date)->subDays(6) <= Carbon::now() &&
```

Comment:
The magic number 6 has no relationship to the comment - please explain why in a comment subDays(6) is used since it isn't self-explanatory.